examples/src/bin/gif_paintable.rs

```diff
@@ -0,0 +1,305 @@
+// A custom GDK paintable capable of rendering a GIF
+// The paintable makes uses of the awesome image
+// crate to read a gif file and transform it to a Vec<Frame>
```

Comment:
That sounds like a bad idea, especially if the GIF has loops. You can easily use up all memory like this :)

It would be better to decode on demand.

Reply:
> That sounds like a bad idea, especially if the GIF has loops. You can easily use up all memory like this :)
> 
> It would be better to decode on demand.

Ah that sounds better, but for now I have to figure out first why my texture isn't exactly what I'm supposed to get